requirements.txt

```diff
@@ -1 +1 @@
-PyYAML
+PyYAML==5.3.1
```

Comment:
I think this is being too strict. Unless we know something breaks, I don't see a need to specify the version at all here (and when we know something breaks, we should avoid the breakage rather than pinning versions). In particular, 5.3.1 isn't what's shipped in any ubuntu before focal.

Reply:
After a second thought, unpinnned them all.